sys-apps/ipmicfg/ipmicfg-1.27.1.170901.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit versionator
+
+MY_DATE="$(get_version_component_range 4)"
+MY_PN="${PN^^}"
+MY_PV="$(get_version_component_range 1-3)"
+
+DESCRIPTION="IPMICFG is an in-band utility for configuring IPMI devices"
+HOMEPAGE="https://www.supermicro.com"
+SRC_URI="ftp://ftp.supermicro.com/utility/${MY_PN}/${MY_PN}_${MY_PV}_build.${MY_DATE}.zip"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="supermicro"
+SLOT="0"
+
+RDEPEND="sys-libs/glibc"
+
+RESTRICT="bindist fetch mirror strip"
+
+S="${WORKDIR}/${MY_PN}_${MY_PV}_build.${MY_DATE}"
+
+QA_PREBUILT="Linux/32bit/IPMICFG-Linux.x86 Linux/64bit/IPMICFG-Linux.x86_64"
+
+pkg_nofetch() {
+	elog "Please download ${A} from"
+	elog "${HOMEPAGE}/SwDownload/UserInfo.aspx?sw=0&cat=IPMI"
```

Comment:
Please don't ref HOMEPAGE like this. It is multi-value, and people may want to add extra sites to it.

Reply:
What to do instead? Write the full plain URL instead of a variable?

Reply:
Yes.

Reply:
URL is now in plain text by newest push.